ideascube/monitoring/views.py

```diff
@@ -159,9 +159,9 @@ class InventoryDelete(DeleteView):
 class InventoryExport(CSVExportMixin, DetailView):
     model = Inventory
 
-    def get(self, request, *args, **kwargs):
+    def get(self, *args, **kwargs):
         self.object = self.get_object()
```

Comment:
There's more I want to do to the `CSVExportMixin` to simplify it even more.

I had started, but hurt myself to this `InventoryExport` view being a bit weird: its model is the `Inventory` but the items it exports are `Specimen` instances. Other views deriving from the `CSVExportMixin` don't have this peculiarity, so I couldn't easily just do the changes I had in mind.

But I'll get back to it eventually. :wink: 

Therefore, I'd request we leave this view as it is for now. Works for you?